14_more_sinatra/badreads/app/views/genres/edit.erb

```diff
@@ -0,0 +1,18 @@
+<form action="/genres/<%= @genre.id %>" method="post">
+  <input type="hidden" name="_method" value="patch" />
+  <input type="text" name="name" placeholder="Name" value="<%= @genre.name %>"/>
+  <div>
+    <% @books.each do |b| %>
+    <input type="checkbox" id="<%= b.id %>" name="book_ids" value="<%= b.id %>"
+      <% if @genre.book_ids.include?(b.id) %>checked<% end %>/>
```

Comment:
remember, we saw this before. it's exactly the same as what I did for the radio buttons in `books/new.erb`